diff --git a/node_modules/react-native-gifted-chat/lib/Avatar.js b/node_modules/react-native-gifted-chat/lib/Avatar.js
index 50b7df5..b79a050 100644
--- a/node_modules/react-native-gifted-chat/lib/Avatar.js
+++ b/node_modules/react-native-gifted-chat/lib/Avatar.js
@@ -11,8 +11,8 @@ const styles = {
             alignSelf: 'flex-start',
         },
         image: {
-            height: 36,
-            width: 36,
+            height: 41,
+            width: 41,
             borderRadius: 18,
         },
     }),
@@ -24,8 +24,8 @@ const styles = {
             alignSelf: 'flex-start',
         },
         image: {
-            height: 36,
-            width: 36,
+            height: 41,
+            width: 41,
             borderRadius: 18,
         },
     }),
diff --git a/node_modules/react-native-gifted-chat/lib/Message/index.js b/node_modules/react-native-gifted-chat/lib/Message/index.js
index a239261..18a228a 100644
--- a/node_modules/react-native-gifted-chat/lib/Message/index.js
+++ b/node_modules/react-native-gifted-chat/lib/Message/index.js
@@ -127,7 +127,7 @@ export const Message = (props) => {
         {messageContent}
       </View>);
     return (<View onLayout={onMessageLayout}>
-      <ReanimatedSwipeable ref={swipeableRef} friction={2} overshootFriction={8} renderRightActions={swipeToReplyDirection === 'left' ? renderSwipeAction : undefined} renderLeftActions={swipeToReplyDirection === 'right' ? renderSwipeAction : undefined} onSwipeableOpen={handleSwipeableOpen} onSwipeableWillOpen={handleSwipeableWillOpen}>
+      <ReanimatedSwipeable ref={swipeableRef} friction={2} overshootFriction={8} dragOffsetFromLeftEdge={swipeToReplyDirection === 'right' ? 10 : Number.MAX_SAFE_INTEGER} dragOffsetFromRightEdge={swipeToReplyDirection === 'left' ? 10 : Number.MAX_SAFE_INTEGER} renderRightActions={swipeToReplyDirection === 'left' ? renderSwipeAction : undefined} renderLeftActions={swipeToReplyDirection === 'right' ? renderSwipeAction : undefined} onSwipeableOpen={handleSwipeableOpen} onSwipeableWillOpen={handleSwipeableWillOpen}>
         {messageContent}
       </ReanimatedSwipeable>
     </View>);
diff --git a/node_modules/react-native-gifted-chat/lib/MessagesContainer/index.js b/node_modules/react-native-gifted-chat/lib/MessagesContainer/index.js
index 14406fc..ada8315 100644
--- a/node_modules/react-native-gifted-chat/lib/MessagesContainer/index.js
+++ b/node_modules/react-native-gifted-chat/lib/MessagesContainer/index.js
@@ -48,7 +48,7 @@ export const MessagesContainer = (props) => {
             return;
         if (isVisible)
             setIsScrollToBottomVisible(true);
-        scrollToBottomOpacity.value = withTiming(isVisible ? 1 : 0, { duration: 250 }, isFinished => {
+        scrollToBottomOpacity.value = withTiming(isVisible ? 1 : 0, { duration: 0 }, isFinished => {
             if (isFinished && !isVisible)
                 runOnJS(setIsScrollToBottomVisible)(false);
         });
@@ -158,9 +158,9 @@ export const MessagesContainer = (props) => {
             return null;
         if (!isScrollToBottomVisible)
             return null;
-        return (<Pressable style={[styles.scrollToBottom, scrollToBottomStyle]} onPress={handleScrollToBottomPress}>
+        return (<View style={[styles.scrollToBottom, scrollToBottomStyle]}>
         {scrollToBottomContent}
-      </Pressable>);
+      </View>);
     }, [isScrollToBottomEnabled, isScrollToBottomVisible, handleScrollToBottomPress, scrollToBottomContent, scrollToBottomStyle]);
     const onLayoutList = useCallback((event) => {
         listHeight.value = event.nativeEvent.layout.height;
diff --git a/node_modules/react-native-gifted-chat/lib/Send.js b/node_modules/react-native-gifted-chat/lib/Send.js
index b7b8236..6726dc9 100644
--- a/node_modules/react-native-gifted-chat/lib/Send.js
+++ b/node_modules/react-native-gifted-chat/lib/Send.js
@@ -17,7 +17,7 @@ export const Send = ({ text, containerStyle, children, textStyle, label = 'Send'
     }, [text, onSend, isTextOptional]);
     const isVisible = useMemo(() => isSendButtonAlwaysVisible || !!text?.trim().length, [isSendButtonAlwaysVisible, text]);
     useEffect(() => {
-        opacity.value = withTiming(isVisible ? 1 : 0, { duration: 200 });
+        opacity.value = withTiming(isVisible ? 1 : 0, { duration: 0 });
     }, [isVisible, opacity]);
     const animatedStyle = useAnimatedStyle(() => ({
         opacity: opacity.value,
diff --git a/node_modules/react-native-gifted-chat/lib/components/ReplyPreview.js b/node_modules/react-native-gifted-chat/lib/components/ReplyPreview.js
index 6aeb6d7..3cb1a66 100644
--- a/node_modules/react-native-gifted-chat/lib/components/ReplyPreview.js
+++ b/node_modules/react-native-gifted-chat/lib/components/ReplyPreview.js
@@ -2,7 +2,7 @@ import React, { useEffect } from 'react';
 import { Image, Pressable, StyleSheet, Text, View, } from 'react-native';
 import Animated, { useAnimatedStyle, useSharedValue, withTiming, Easing, interpolate, runOnJS, } from 'react-native-reanimated';
 import { useColorScheme } from '../hooks/useColorScheme';
-const ANIMATION_DURATION = 200;
+const ANIMATION_DURATION = 0;
 const ANIMATION_EASING = Easing.bezier(0.25, 0.1, 0.25, 1);
 const DEFAULT_HEIGHT = 68;
 const styles = StyleSheet.create({
diff --git a/node_modules/react-native-gifted-chat/lib/components/TouchableOpacity.js b/node_modules/react-native-gifted-chat/lib/components/TouchableOpacity.js
index a50cdbf..56e00da 100644
--- a/node_modules/react-native-gifted-chat/lib/components/TouchableOpacity.js
+++ b/node_modules/react-native-gifted-chat/lib/components/TouchableOpacity.js
@@ -6,16 +6,16 @@ export const TouchableOpacity = ({ children, style, activeOpacity = 0.2, onPress
     const opacity = useSharedValue(1);
     const isAnimationInFinished = useSharedValue(false);
     const handlePressIn = useCallback(() => {
-        opacity.value = withTiming(activeOpacity, { duration: 150 }, () => {
+        opacity.value = withTiming(activeOpacity, { duration: 0 }, () => {
             isAnimationInFinished.value = true;
         });
     }, [activeOpacity, opacity, isAnimationInFinished]);
     const handlePressOut = useCallback(() => {
         setTimeout(() => {
             'worklet';
-            opacity.value = withTiming(1, { duration: 150 });
+            opacity.value = withTiming(1, { duration: 0 });
             isAnimationInFinished.value = false;
-        }, isAnimationInFinished.value ? 0 : 150);
+        }, 0);
     }, [opacity, isAnimationInFinished]);
     const handleActiveStateChange = useCallback((isActive) => {
         if (isActive)
